src/utils/constants.js

```diff
@@ -3,7 +3,7 @@ import {noView} from 'aurelia-framework';
 /**
  * @type {string}
  */
-export const APP_NAME = 'iViewer';
+export const APP_NAME = 'iviewer';
```

Comment:
we could, I didn't do so because the main reason for this PR was to keep the app name fixed (url). So I didn't bother passing yet another thing from the plugin.